compiler+runtime/test/bash/module/clojure-test/pass-test

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
1. You've put these tests under the `module` dir, but they are unrelated to module loading; moving this up to be `test/bash/clojure.test` should be fine.
2. Why use `clojure-test` instead of `clojure.test`?

Reply:
That makes a lot more sense, I misunderstood the convention.